torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -133,6 +131,10 @@ class ZeroRedundancyOptimizer(Optimizer):
         >>> ddp(inputs).sum().backward()
         >>> opt.step()
 
+    .. note: Currently, ``ZeroRedundancyOptimizer`` only supports using
+        the same dtype for all parameters and does not support model
+        parallelism.
```

Comment:
does model parallelism here refer to input parameters residing on different devices? If so, would it be good to change the wording to indicate that (wondering if we can use `ZeRO` in general model parallel applications where the parameters passed into it are on the same device).

Reply:
Yes, I meant "model parallelism" to mean input parameters residing on different devices. I have rephrased the wording to make it (hopefully) clear.